### PR TITLE
fix: manifest.json

### PR DESCRIPTION
**Pull request by aybarsayan**
## Checklist:
Changed the manifest version to v3

- [x] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

**Review by Dudleyneedham — COMMENTED** [review]
Some questions about what these things do

File: src/static/manifest.json [hunk 1]
```diff
@@ -41,8 +38,20 @@
     "clipboardWrite",
     "storage"
   ],
-  "content_security_policy": "script-src 'self' 'wasm-eval'; object-src 'self';",
+  "host_permissions": [
+    "*://*/*"
```
[the remark this answers is not in the document]
Comment (aybarsayan) [reply src/static/manifest.json:42]: Chrome has decided not to fully support WASM for extensions in its new versions. I got quite stuck on this, but later, when I saw that this solution was implemented in the Polkadot.js Chrome extension, I integrated it myself, and the problem was resolved.

**Comment by arty-name** [comment]
There was [another branch for this purpose](https://github.com/BTE-Trusted-Entity/sporran-extension/commit/1ad75fb20027e56037921a70362754c30e3fc555) since a long time, I believe a few more changes are necessary

File: src/static/manifest.json [hunk 2]
```diff
@@ -41,8 +38,20 @@
...
+      "resources": [
+        "js/injectedScript.js"
+      ],
+      "matches": [
+        "*://*/*"
+      ]
+    }
   ]
-}
+}
```
Comment (arty-name) [inline src/static/manifest.json:57]: I highly recommend having the EditorConfig extension for the automatic configuration of basic formatting, like the trainling newline. This extension is a must for all BTE projects.

Comment (aybarsayan) [reply src/static/manifest.json]: Fixed at [4cb8865](https://github.com/BTE-Trusted-Entity/sporran-extension/pull/2632/commits/4cb8865be70f9645c14ee2fdee6ea32d2f536a03)

**Review by arty-name — APPROVED** [review]

**Review by Dudleyneedham — APPROVED** [review]
LGTM

**Pull request merged**